packages/editor/src/components/post-visibility/index.js

```diff
@@ -75,7 +75,7 @@ export class PostVisibility extends Component {
 		};
 
 		return [
-			<fieldset key="visibility-selector">
+			<fieldset key="visibility-selector" className="editor-post-visibility">
```

Comment:
For what purpose was this change made? It doesn't seem to be used.

Reply:
Hi @aduth, I think this class was added as a way to target this element with a high specificity in order to avoid the usage of "!important" https://github.com/WordPress/gutenberg/pull/9138/files/48b561ade1dfd50704d6fff08ef55721b282b5bf#diff-fbd97d3615849017cce9d16a546f9fd3.
That's also the reason why we are nesting it inside .edit-post-post-visibility__dialog maybe we should have added a comment describing this.

Reply:
I modified this selector in #9800 and added a note about the nesting in my PR description